hack/format-scripts.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Picking up @ahmetb's comment from https://github.com/kubernetes-sigs/krew/issues/332#issuecomment-528600895, we should probably remove this reformat script. For example, there is also no script for `goimports -w` or similar. Instead, I recommend documenting the shell linter in `docs/CONTRIBUTOR_GUIDE.md`.

Reply:
Agreed.

Reply:
Also The error message in the validation script should tell the committer what command to run to fix, and where the errors were found (or a diff).

Reply:
Done, I guess. Please, give another look. Thanks!